### ISSUE TITLE
False positive - Non-optimal if conditions

### ISSUE THREAD

**Issue by 3DFace** [kalessil/phpinspectionsea#356]
Code to type:

```php
$a = foo();
if($a === null){
	$a = bar();
	if($a === null){ // False positive here, cause $a has been changed
		echo 'Hey!';
	}
}
```

**Comment by kalessil** [comment]
Thank you for reporting @3DFace, but this duplicates #339
